apps/evisa/behaviours/limit-documents.js

```diff
@@ -0,0 +1,16 @@
+module.exports = superclass => class LimitDocs extends superclass {
+  validate(req, res, next) {
+    const images = req.sessionModel.get('images');
+    if (images && images.length >= 2) { // TODO replace magic number with config
```

Comment:
I set the limit to 2 for testing, should be updated to what is required from BA

Reply:
If it's a hard-coded value I'll give it the `const` treatment